pkg/backend/httpstate/environments.go

```diff
@@ -73,6 +74,6 @@ func (b *cloudBackend) OpenYAMLEnvironment(
 	if err != nil || len(diags) != 0 {
 		return nil, convertESCDiags(diags), err
 	}
-	env, err := b.escClient.GetOpenEnvironment(ctx, org, "yaml", id)
+	env, err := b.escClient.GetOpenEnvironmentWithProject(ctx, org, "project", "yaml", id)
```

Comment:
@pgavlin It seems like it doesn't matter what we pass as project/env when we pass an open session id. is that correct?

Reply:
Just for posterity how does the hardcoding these values work?

Reply:
The gist is that these values don't really matter because nothing is actually being created as this is an anonymous environment. The placeholder env name is `"yaml"` and I made the creative decision to name the placeholder project `"project"`